Fixed Hassbian 404 URLs

leytpapas opened the pull request:
.md were moved from 'hassbian-scripts/blob/master/docs' to 'hassbian-scripts/tree/dev/docs/suites'

**Description:**

Fixes: #9700

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9755"><img src="https://gitpod.io/api/apps/github/pbs/github.com/leytpapas/home-assistant.io.git/3afffb1592d05e32b2fad812ffde3e4456461905.svg" /></a>

leytpapas committed:
Fixed URLs

.md were moved from 'hassbian-scripts/blob/master/docs' to 'hassbian-scripts/tree/dev/docs/suites'

frenck commented on source/_docs/installation/hassbian/customization.markdown:
```diff
@@ -16,26 +16,26 @@ The tool is available by running `hassbian-config`.
- - [Mosquitto](https://github.com/home-assistant/hassbian-scripts/blob/master/docs/mosquitto.md)
- - [Samba](https://github.com/home-assistant/hassbian-scripts/blob/master/docs/samba.md)
- - [Webterminal](https://github.com/home-assistant/hassbian-scripts/blob/master/docs/webterminal.md)
+ - [AppDaemon](https://github.com/home-assistant/hassbian-scripts/tree/dev/docs/suites/appdaemon.md)
```
I think we should link them to the `master` branch, not the `dev` branch.

leytpapas committed:
Fixed Hassbian 404 URLs

Also updated the list with more available scripts. Homebridge was not found in the hassbian-scripts repository so it was deleted
